tokenizer/src/main/scala/anorm/TokenizedStatement.scala

```diff
@@ -41,7 +44,7 @@ private[anorm] object TokenizedStatement {
 
   @annotation.tailrec
   private[anorm] def tokenize[T](ti: Iterator[String], tks: List[StatementToken], parts: Seq[String], ps: Seq[T with Show], gs: List[TokenGroup], ns: List[String], m: Map[String, T]): (TokenizedStatement, Map[String, T]) = if (ti.hasNext) ti.next match {
-    case "%" => tokenize(ti, PercentToken :: tks, parts, ps, gs, ns, m)
```

Comment:
So does this change means that `%` is no longer a special character for Anorm?

Also, did you mean to delete the code instead of just commenting it out?